### PR TITLE
ci(github): fix checklist PR comment

### PR DESCRIPTION
**Pull request by lahabana**
## Supporting documentation

xrel: #11654 #11666

### CODE REVIEW

**Review by lukidzi — APPROVED** [review]

File: .github/workflows/check.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-name: "PR health"
```
Comment (michaelbeaumont) [inline .github/workflows/check.yaml:1]: Why this change?

Comment (lahabana) [reply .github/workflows/check.yaml:1]: Because the title makes debugging just harder:

Comment (michaelbeaumont) [reply .github/workflows/check.yaml:1]: @lahabana i think your image is missing

Comment (michaelbeaumont) [reply .github/workflows/check.yaml:1]: Does it? That's so noisy to me. the awful prefix in front of everything and then `.yaml`. Can't you always jump to Workflow file on the left of the run summary and even see the name at the top of the list of runs for that workflow?
